ext/EnzymeExt.jl

```diff
@@ -93,6 +95,65 @@ module EnzymeExt
 
         # TODO in KA backends like CUDAKernels, etc have a version with a parent job type
         TapeType = EnzymeCore.tape_type(ReverseSplitModified(ReverseSplitWithPrimal, ModifiedBetween), FT, Const,  Const{ctxTy}, map(Core.Typeof, args2)...)
+        @show TapeType
+
+
+        subtape = Array{TapeType}(undef, size(blocks(iterspace)))
```

Comment:
This shouldn't be Array, but it should be the device type specific to the array.

Is there a way to get that @vchuravy @michel2323

Reply:
KernelAbstractions.allocate